database/modules/user.model.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
+const joi = require("joi");
+const jsonWebToken = require("jsonwebtoken");
+const config = require("../../config.env");
+const { Timestamp, ObjectId } = require("mongodb");
+
+const UserSchema = new mongoose.Schema(
+  {
+    name: { type: String, required: true },
+    email: {
+      type: String,
+      unique: true,
+      validate: {
+        validator(email) {
+          const { error } = joi.string().email().validate(email);
+
+          if (error) throw new Error("Email not vlid");
+        },
+      },
+    },
+    passwordHash: { type: String },
+    familyId: { type: ObjectId, default: null },
+    tokens: [
+      {
+        token: { type: String, require: true },
```

Comment:
Look, as far as I remember than option called `required` rather than `require`. Be careful with options naming